web-server/views/manager/prosumers.ejs

```diff
@@ -42,34 +41,38 @@
           </tr>
         </thead>
         <tbody id="prosumerTable">
-          <% for (var p in prosumers) { %>
+          <% for (var i in data.prosumers) { %>
+          <% var prosumer = data.prosumers[i]; %>
+          <% var house = data.houses[i]; %>
           <tr>
-            <td id="<%= prosumers[p].id %>.online"></td>
-            <td><%- include('../partials/avatar', {user: prosumers[p], size: 30})%></td>
-            <td id="<%= prosumers[p].id %>.name"></td>
-            <td id="<%= prosumers[p].id %>.email"></td>
-            <td id="<%= prosumers[p].id %>.removed"></td>
-            <td id="<%= prosumers[p].id %>.created_at"></td>
-            <td id="<%= prosumers[p].id %>.blocked"></td>
-            <td id="<%= prosumers[p].id %>.blackOut"></td>
-            <td>
+            <td class="<%= prosumer.id %> online">...</td>
+            <td><%- include('../partials/avatar', {user: prosumer, size: 30})%></td>
+            <td class="<%= prosumer.id %> name"><%= prosumer.name %></td>
+            <td class="<%= prosumer.id %> email"><%= prosumer.email %></td>
+            <td class="<%= prosumer.id %> removed"><%= prosumer.removed ? "Yes" : "No" %></td>
+            <td class="<%= prosumer.id %> created_at"><%= prosumer.createdAt %></td>
```

Comment:
I removed the ejs here before because the timestamp was displayed differently from the js script. I think we should remove the ejs if the timestamp is displayed differently, to make it more consistent.

Reply:
We probably don't need to refresh the page when blocking, I will add ajax request instead of form submit.